### PR TITLE
2

### PR DESCRIPTION
**Pull request by glor-n**

### CODE REVIEW

File: jupyter_book/shared_bibliography/F.Lacquaniti_and_C.Maioli.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+## Tags and Keywords 
+```
+[what species was studied? What tools were used? What kind of data did they get? What was the main topic?]
+```
+
+#postural-control  
+#limb-position  
+#cat-posture  
+#kinetic  
+#geometric-parameters
```
Comment (sahana-dhar) [inline jupyter_book/shared_bibliography/F.Lacquaniti_and_C.Maioli.md:44]: love the specificity

File: jupyter_book/shared_bibliography/F.Lacquaniti_and_C.Maioli.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+## Brief Summary
+```
+ 1. What did they do (what activities)? 
+ 2. What did they measure (what data)?
+ 3. What did they find (what results)?
+ 4. What does it mean (what new knowledge)?
+```
+1. cat on platform while weight distribution, platform angle, and load attachment (center of mass) were changed
+2. limb positions and contact fores
+3. limb geometry was reserved unchaged, whereas kinetic parameters were severely affected when the load has limited contact but when the load has contact with most of the body it causes irregular balance and limb geomentry is affected
```
Comment (sahana-dhar) [inline jupyter_book/shared_bibliography/F.Lacquaniti_and_C.Maioli.md:21]: great summary

File: jupyter_book/shared_bibliography/F.Lacquaniti_and_C.Maioli.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# (Lacquaniti, F., & Maioli, C., 1994)
+
+## Citation (APA Format)
+
+Lacquaniti, F., & Maioli, C. (1994). Independent control of limb position and contact forces in cat posture. Journal of neurophysiology, 72(4), 1476–1495. https://doi.org/10.1152/jn.1994.72.4.1476
+
+## Citation Nature Format
+
```
Comment (ShreyaSrii) [inline jupyter_book/shared_bibliography/F.Lacquaniti_and_C.Maioli.md:8]: great citation formatting!

File: jupyter_book/shared_bibliography/F.Lacquaniti_and_C.Maioli.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+## Method figure
+```
+copy paste the main methods figure, if there is one]
+```
+![img.png](images/img.png)
+
+## Main Results figure
+`[copy the (or one of the) main results figure ]`
+![img_1.png](images/img_1.png)
```
Comment (sahana-dhar) [inline jupyter_book/shared_bibliography/F.Lacquaniti_and_C.Maioli.md:32]: i guess i'll imagine these images

**Review by sahana-dhar — APPROVED** [review]

File: jupyter_book/shared_bibliography/F.Lacquaniti_and_C.Maioli.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+```
+ 1. What did they do (what activities)? 
+ 2. What did they measure (what data)?
+ 3. What did they find (what results)?
+ 4. What does it mean (what new knowledge)?
+```
+1. cat on platform while weight distribution, platform angle, and load attachment (center of mass) were changed
+2. limb positions and contact fores
+3. limb geometry was reserved unchaged, whereas kinetic parameters were severely affected when the load has limited contact but when the load has contact with most of the body it causes irregular balance and limb geomentry is affected
+4. independent control of limb geometry and contact forces
```
Comment (madelynredick) [inline jupyter_book/shared_bibliography/F.Lacquaniti_and_C.Maioli.md:22]: maybe elaborate some more on the results (#4)      <3 <3 <3

File: jupyter_book/shared_bibliography/F.Lacquaniti_and_C.Maioli.md (hunk 2 above)
Comment (glor-n) [reply jupyter_book/shared_bibliography/F.Lacquaniti_and_C.Maioli.md:21]: thanks queen

**Review by jonmatthis — APPROVED** [review]

**Pull request merged**